catalog/transformer/catalog-transformer-html/src/main/resources/templates/category.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="metacard-category">
+  <h3 class="category-title">{{ title }}</h3>
+  <table class="category-table">
+    <tr>
+      <th class="attribute-name">Property</th>
+      <th class="attribute-value">Value</th>
+    </tr>
+    {{#each attributeMappings}}
+      <tr class="metacard-attribute">
+        <td class="attribute-name">{{ @key }}</td>
+
+        {{#isEmptyValue}}
```

Comment:
Consider using polymorphism for this. Instead of the template detecting the "type" of an item and acting accordingly, each item could describe how it should be displayed and the template wouldn't have to have any decision-making logic.

Reply:
I went to implement polymorphism for value models this morning and ran into a design dilemma. If I remove these custom if helpers then I remove the ability for the template to define the different structure for each type of value model since different value models (like the media model) wrap the data in different HTML tags.

Reply:
Another potential solution would be to render a template in each `HtmlValueModel` and stub in a section of HTML that's stored as a string in each `HtmlValueModel`. This would be a good middle ground. Thoughts?

Reply:
I ran into a snag while making this change. If I store a value model template in a string it escapes the HTML. Even if I unescape the HTML using `StringEscapeUtils.unescapeHtml()` somewhere along the way Handlebars escapes it again. 

Unless you see a way forward I think I will leave it as it was.

Reply:
Just implemented polymorphism for value models. The handlebars template is now much cleaner